kubeflow/trainer/backends/localprocess/job.py

```
@@ -0,0 +1,184 @@
+                return
+
+        current_dir = os.getcwd()
+        try:
+            self._start_time = datetime.now()
+            _c = " ".join(self.command)
+            logger.debug(f"[{self.name}] Started at {self._start_time} with command: \n {_c}")
+
+            # change working directory to venv before executing script
+            os.chdir(self.execution_dir)
+
+            self._process = subprocess.Popen(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)